dataactbroker/helpers/validation_helper.py

```diff
@@ -427,6 +450,49 @@ def check_length(data, length_fields, report_headers, csv_schema, short_cols, fl
     return errors
 
 
+def check_field_format(data, format_fields, report_headers, short_cols, flex_data):
+    """ Check if all fields that are a type other than string match that type.
```

Comment:
Can we update this description to something like "Check if all the fields are in the appropriate format"? Similarly, with the other point, is this going to check the formatting of all types or just dates (considering the error message, rule label, and expected value are pretty geared towards the date format). If so, can we rename to `check_date_field_format` and the description to "Check if all date fields are in the appropriate format"?